nabo/nabo.h

```diff
@@ -256,7 +258,7 @@ namespace Nabo
 		typedef typename Eigen::Matrix<Index, Eigen::Dynamic, 1> IndexVector;
 		//! a matrix of indices to data points
 		typedef typename Eigen::Matrix<Index, Eigen::Dynamic, Eigen::Dynamic> IndexMatrix;
-		
```

Comment:
Some remark as before on whitespaces, even if we do decide they should be trimmed, a patch concerned with boost dependency removal should change the rest of the code if not necessary.